src/main/index.ts

```diff
@@ -55,6 +56,7 @@ app.on("ready", async () => {
     clusterStore.load(),
     workspaceStore.load(),
     extensionsStore.load(),
+    filesystemProvisionerStore.load(),
```

Comment:
Should `filesystemProvisionerStore` load also in renderer?

Reply:
Yes probably.

Reply:
Done